source/tutorials/nvidia.rst

```diff
@@ -192,17 +213,25 @@ Install the NVIDIA drivers
 
       sudo swupd repair --quick --bundles=lib-opengl
 
-.. note::
+   .. warning:: 
 
-   The NVIDIA software places some files under the :file:`/usr` subdirectory
-   that are not managed by |CL| and conflict with the |CL| stateless design.
+      Although a limited version of :command:`swupd repair` is run above,
+      other uses of the :command:`swupd repair` command should be avoided
```

Comment:
Revise sentence to start with: Other uses of the :command:`swupd repair` ...

Reply:
I wrote it like this intentionally because I found it a little offputting having the steps essentially read 

- Run this command.
- Don't run this command.

Reply:
I understand your approach to this. We can address wording in a later TCS editorial PR.